web/src/OverviewActionBar.tsx

```diff
@@ -659,6 +660,88 @@ export function OverviewWidgets(props: { buttons?: UIButton[] }) {
     </WidgetRoot>
   )
 }
+const EnableDisableResourceRoot = styled(ButtonRoot)<{
+  confirmingDisable: boolean
+}>`
+  ${(props) =>
+    props.confirmingDisable &&
+    `
+    color: ${Color.white};
+    background-color: ${Color.red};
+
+    &:hover {
+      color: ${Color.white};
+      background-color: ${Color.red};
+    }
+  `};
+  margin-left: auto;
+  padding-left: ${SizeUnit(0.25)};
+`
+const EnableDisableResourceButtonContent = styled.div`
+  .MuiIcon-root {
+    font-size: ${FontSize.small};
```

Comment:
(design nitpick) how about adding `padding-right: ${SizeUnit(1 / 8)}` to the icon, so there's a little space between the icon and button text?